dspace-server-webapp/src/test/java/org/dspace/app/rest/AuthorizationRestRepositoryIT.java

```diff
@@ -1517,6 +1525,873 @@ public void findByObjectAndFeatureInternalServerErrorTest() throws Exception {
+            .param("type", "core.community")
+            .param("uuid", comId)
+            .param("uuid", secondComId)
+            .param("projection", "level")
+            .param("embedLevelDepth", "1")
+            .param("feature", alwaysFalse.getName())
+            .param("feature", trueForAdmins.getName())
+            .param("feature", alwaysTrue.getName()))
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("$.page.totalElements", is(2)))
+            .andExpect(jsonPath("$._embedded.authorizations", containsInAnyOrder(
+                allOf(
+                    hasJsonPath("$.id", is(
+                        alwaysTrue.getName() + "_"
+                            + comRest.getUniqueType() + "_" + comRest.getId()
+                    )),
+                    hasJsonPath("$.type", is("authorization")),
+                    hasJsonPath("$._embedded.feature.id", is(alwaysTrue.getName())),
+                    hasJsonPath("$._embedded.eperson", nullValue())
+                ),
+                allOf(
+                    hasJsonPath("$.id", is(
+                        alwaysTrue.getName() + "_"
+                            + secondComRest.getUniqueType() + "_" + secondComRest.getId()
+                    )),
+                    hasJsonPath("$.type", is("authorization")),
+                    hasJsonPath("$._embedded.feature.id", is(alwaysTrue.getName())),
+                    hasJsonPath("$._embedded.eperson", nullValue())
+                )
+            )));
+    }
+
+    @Test
+    /**
+     * Verify that the search by many objects and features works return 204 No Content when no feature is granted.
+     *
+     * @throws Exception
+     */
+    public void findByMultipleObjectsAndFeatureNotGrantedTest() throws Exception {
+
+        context.turnOffAuthorisationSystem();
+
+        Community community = CommunityBuilder.createCommunity(context).build();
+        Collection collection = CollectionBuilder.createCollection(context,community).build();
+        Item item = ItemBuilder.createItem(context, collection).build();
+        String itemId = item.getID().toString();
+        ItemRest itemRest = itemConverter.convert(item, DefaultProjection.DEFAULT);
+        Item secondItem = ItemBuilder.createItem(context, collection).build();
+        String secondItemId = secondItem.getID().toString();
+        ItemRest secondItemRest = itemConverter.convert(secondItem, DefaultProjection.DEFAULT);
+
+        context.restoreAuthSystemState();
+
+        String adminToken = getAuthToken(admin.getEmail(), password);
+
+        // verify that it works for administrators - with eperson parameter
+        getClient(adminToken).perform(get("/api/authz/authorizations/search/objects")
+            .param("type", "core.item")
+            .param("uuid", itemId)
+            .param("uuid", secondItemId)
+            .param("feature", alwaysFalse.getName())
+            .param("feature", alwaysFalse.getName())
```

Comment:
You've duplicated `alwaysFalse` in several of these calls (this and next two).  It seems like we only need to pass it once?

Reply:
@tdonohue It is to test the case of passing 2 features for which the user is not allowed, and check that the endpoint returns a proper response.